tests/PhpSpreadsheetTests/Calculation/ArrayTest.php

```diff
@@ -7,10 +7,33 @@
 use PhpOffice\PhpSpreadsheet\Calculation\Calculation;
 use PhpOffice\PhpSpreadsheet\Calculation\Functions;
 use PhpOffice\PhpSpreadsheet\Spreadsheet;
+use PhpOffice\PhpSpreadsheet\Worksheet\Worksheet;
 use PHPUnit\Framework\TestCase;
 
 class ArrayTest extends TestCase
 {
+    private string $originalArrayReturnType;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
```

Comment:
Are this and the corresponding statement in tearDown needed? I'd rather you omitted them if not.